arangod/Aql/SortingGatherExecutor.cpp

```diff
@@ -73,10 +73,16 @@ class OurLessThan {
       } else {
         // Take attributePath into consideration:
         bool mustDestroyA;
-        AqlValue aa = lhs.get(_resolver, attributePath, mustDestroyA, false);
+        AqlValue aa = lhs.get(_resolver,
+                              std::vector<std::string_view>{
+                                  attributePath.begin(), attributePath.end()},
+                              mustDestroyA, false);
         AqlValueGuard guardA(aa, mustDestroyA);
         bool mustDestroyB;
-        AqlValue bb = rhs.get(_resolver, attributePath, mustDestroyB, false);
+        AqlValue bb = rhs.get(_resolver,
+                              std::vector<std::string_view>{
+                                  attributePath.begin(), attributePath.end()},
+                              mustDestroyB, false);
```

Comment:
We should not do this here.
This is a comparator, which can be called millions of times in a single query. It will be prohibitively expensive to create a new vector with heap allocations every time we look up an attribute in a document.